bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueSensorHandler.java

```diff
@@ -227,15 +224,23 @@ public void handleConfigurationUpdate(Map<String, Object> configurationParameter
     }
 
     @Override
-    public void onSensorStateChanged(@Nullable HueBridge bridge, FullSensor sensor) {
+    public boolean onSensorStateChanged(@Nullable HueBridge bridge, FullSensor sensor) {
         logger.trace("onSensorStateChanged() was called");
 
+        // TODO Maybe this check is obsolete
```

Comment:
With your changes, this check becomes useless (it was useful before).

Reply:
Removed